mmtrack/apis/inference.py

```diff
@@ -40,8 +40,9 @@ def init_model(config, checkpoint=None, device='cuda:0', cfg_options=None):
         if 'meta' in checkpoint and 'CLASSES' in checkpoint['meta']:
             model.CLASSES = checkpoint['meta']['CLASSES']
     if not hasattr(model, 'CLASSES'):
-        if hasattr(model, 'detector') and hasattr(model.detector, 'CLASSES'):
-            model.CLASSES = model.detector.CLASSES
+        if hasattr(model, 'detector'):
+            # some detectors may don't have `CLASSES`.
+            model.CLASSES = getattr(model.detector, 'CLASSES', None)
```

Comment:
why change like this?

Reply:
The detector of `deepsort` model we provided doesn't have the `CLASSES` attribute.

Reply:
The original logic of codes can guarantee it.
Besides, The original logic will print a warning when `hasattr(model.detector, 'CLASSES') is False`, while the modified logic will not. I think it is better to keep the logic unchanged.